cpp/inline/mcts/NNEvaluationServiceParams.inl

```diff
@@ -1,7 +1,5 @@
-#include <mcts/NNEvaluationServiceParams.hpp>
-
 #include <boost/filesystem.hpp>
-
+#include <mcts/NNEvaluationServiceParams.hpp>
```

Comment:
There is one exception to the `#include` grouping + alphabetization rule I described.

When you have `A.cpp` or `A.inl`, the corresponding `A.hpp` should always be the very first `#include` at the top of the file, with a line break after it. Then, you can do the standard alphabetized groupings after that.